app/_includes/md/kic/class.md

```diff
@@ -22,7 +22,7 @@ spec:
   controller: ingress-controllers.konghq.com/kong
 " | kubectl apply -f -
 ```
-Response:
+The results should look like this:
```

Comment:
@Kong/docs-maintainers We don't have consistent text to use when showing a response. Should we standardise across pages?

Reply:
Hi @mheap I updated this based on https://github.com/Kong/docs.konghq.com/pull/6043.

Reply:
Ah! I didn't realise it was in the style guide. Thanks for sharing